spec/controllers/instructeurs/dossiers_controller_spec.rb

```diff
@@ -1516,4 +1516,37 @@
       expect(assigns(:attachments_and_libelles).count).to eq 3
     end
   end
+
+  describe 'dossier_labels' do
+    context 'it create dossier labels' do
+      subject { post :dossier_labels, params: { procedure_id: procedure.id, dossier_id: dossier.id, procedure_label_id: [ProcedureLabel.first.id] }, format: :turbo_stream }
+      it 'works' do
+        subject
+        dossier.reload
+
+        expect(dossier.dossier_labels.count).to eq(1)
+      end
+
+      it { expect(subject.body).to include('header-top') }
```

Comment:
est-ce que ce ne serait pas plus explicite de tester l'inclusion d'un truc en rapport avec le label ajouté , plutôt qu'un élément du dom classique ?

et au passage grouper les 2 `it` vu que les conditions entre les tests ne changent pas (moins de tests = moins de setup de test = suite de test plus rapide)